_data/releases.yml

```diff
@@ -1,10 +1,10 @@
 - title: Production releases
   releases:
+    - date: Nov 25, 2020
+      version: v20.2.2
+      latest: true
     - date: Nov 20, 2020
       version: v20.2.1
-      latest: true
-    - date: Nov 10, 2020
```

Comment:
@rmloveland, this remove the 20.2.0 release notes, which has the full summary of all features in the major release, from the list of release, so there's currently no way to find the page unless you know it exists. Was this intentional? In any case, I'll open a PR to put it back.

Reply:
Definitely human error!